clean-abap/CleanABAP.md

```diff
@@ -4606,6 +4606,15 @@ by redirecting them to test data defined in the unit test without interfering wi
 
 - Use the CDS test environment (`CL_CDS_TEST_ENVIRONMENT`) to test your CDS views.
 
+- Use the AUTHORITY-CHECK test environment (`CL_AUNIT_AUTHORITY_CHECK`) to withdraw authorizations during the runtime of the test.
```

Comment:
I'd rather call it a Helper API or something like that, not an environment... The official documentation calls it an API and not an environment: https://help.sap.com/docs/ABAP_PLATFORM_NEW/c238d694b825421f940829321ffa326a/6500d4d8f89a4743a6c0513d659a475b.html

Reply:
I agree with you, however I'd like to note that the ABAP Doc calls it a "Classic AUTHORITY-CHECK Test Environment" 😉

Reply:
I just reached out to the developer responsible for the class and asked them what the best name would be and whether they could align documentation and the ABAP Doc ;)

Reply:
Just got the reply: It is supposed to be called a Test Helper API. The ABAP Doc will be adapted (I guess with 2311).